src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -347,6 +348,7 @@ def load_arguments(self, _):
         c.argument('workload_runtime', arg_type=get_enum_type(workload_runtimes), default=CONST_WORKLOAD_RUNTIME_OCI_CONTAINER)
         # no validation for aks create because it already only supports Linux.
         c.argument('enable_custom_ca_trust', action='store_true')
+        c.argument('custom_ca_trust_certificates', options_list=["--custom_ca_trust_certificates", "--ca-certs"], is_preview=True, help="path to file containing list of base64 encoded CAs")
```

Comment:
```suggestion
        c.argument('custom_ca_trust_certificates', options_list=["--custom-ca-trust-certificates", "--ca-certs"], is_preview=True, help="path to file containing list of base64 encoded CAs")
```

Use `-` instead of `_`.